clients/pkg/promtail/client/client.go

```diff
@@ -91,6 +98,11 @@ func NewMetrics(reg prometheus.Registerer) *Metrics {
 		Name:      "dropped_entries_total",
 		Help:      "Number of log entries dropped because failed to be sent to the ingester after all retries.",
 	}, []string{HostLabel, TenantLabel, DropReasonLabel})
+	m.mutatedEntries = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Can you add one more metrics to track mutated bytes? similar to `droppedBytes`.

For example.
```
var MutatedBytes = prometheus.NewCounterVec(
	prometheus.CounterOpts{
		Namespace: "promtail",
		Name:      "mutated_bytes_total",
		Help:      "The total number of bytes that have been mutated.",
	},
	[]string{HostLabel, TenantLabel, ReasonLabel}
)

```